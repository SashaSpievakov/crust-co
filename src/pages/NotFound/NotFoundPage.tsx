import { ErrorParagraph } from '@src/styles/Base.styled';
import { FC } from 'react';

import { ButtonLink } from '../../components/UI';
import { Article, Title } from './NotFoundPage.styled';

export const NotFoundPage: FC = () => {
  return (
    <Article>
      <Title>Oops! This page doesn&apos;t exist</Title>
      <ErrorParagraph>
        We couldn&apos;t find the page you were looking for. Please check the
        URL or navigate back to the homepage.
      </ErrorParagraph>
      <ButtonLink link="/">Return Home</ButtonLink>
    </Article>
  );
};
