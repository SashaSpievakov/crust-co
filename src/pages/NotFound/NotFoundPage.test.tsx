import '@testing-library/jest-dom';

import { rendererWithProviders } from '../../tests/helpers';
import { NotFoundPage } from './NotFoundPage';

describe('NotFound tests', () => {
  test('should render the NotFound page', () => {
    const snapshot = rendererWithProviders(<NotFoundPage />);
    expect(snapshot).toMatchSnapshot();
  });
});
