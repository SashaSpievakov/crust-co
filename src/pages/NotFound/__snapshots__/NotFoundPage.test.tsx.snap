// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Tests renders the NotFound page 1`] = `
Object {
  "children": Array [
    <h2
      className="sc-fHSyak ehgagf"
    >
      The page wasn't found
    </h2>,
    <p
      className="sc-idXgbr daODfB"
    >
      Make sure you wrote the correct URL
    </p>,
    <a
      className="sc-bcXHqe sc-gswNZR glVyWT bMYeao"
      href="/"
      onClick={[Function]}
    >
      Go Back
    </a>,
  ],
  "props": Object {
    "className": "sc-evzXkX gjeAtR",
  },
  "type": "article",
}
`;
